src/System.ComponentModel.TypeConverter/src/System/ComponentModel/WarningException.cs

```diff
@@ -66,7 +68,8 @@ public WarningException(string message, string helpUrl, string helpTopic)
         /// </summary>
         protected WarningException(SerializationInfo info, StreamingContext context) : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            HelpUrl = (string)info.GetValue("helpUrl", typeof(string));
```

Comment:
These can be GetString with no cast, right? I see other cases when I grep for `(string)info.GetValue`. Just cleanup though: don't hold on that.

Reply:
Correct. I intentionally left them as they are to limit differences to netfx... I can tackle them in a future cleanup PR